test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -155,18 +158,20 @@ Describe "New-Item with links" -Tags @('CI', 'RequireAdminOnWindows') {
         $fileInfo.Target | Should Be $target
         Test-Path $fileInfo.Target | Should be $false
         $fileInfo.LinkType | Should Be "SymbolicLink"
+        $fileInfo.Attributes -band $DirLinkMask | Should Be $SymLinkMask
     }
 
-    It "Should create a symbolic link from directory without error" {
+    It "Should create a symbolic link to directory without error" {
         New-Item -Name $testFolder -Path $tmpDirectory -ItemType directory
         Test-Path $FullyQualifiedFolder | Should Be $true
 
         New-Item -ItemType SymbolicLink -Target $FullyQualifiedFolder -Name $testlink -Path $tmpDirectory
         Test-Path $FullyQualifiedLink | Should Be $true
 
-        $fileInfo = Get-ChildItem $FullyQualifiedLink
+        $fileInfo = Get-Item $FullyQualifiedLink
         $fileInfo.Target | Should Match ([regex]::Escape($FullyQualifiedFolder))
         $fileInfo.LinkType | Should Be "SymbolicLink"
+        $fileInfo.Attributes -band $DirLinkMask | Should Be $DirLinkMask
 
         # Remove the link explicitly to avoid broken symlink issue
         Remove-Item $FullyQualifiedLink -Force
```

Comment:
Ah. I didn't know what "broken symlink issue" the author was referring to, so I didn't remove another dev's code. I am working on a `Remove-Item` issue, but I want to get this in so I can write tests using `New-Item` rather than shelling out a `mklink` command.